docs/examples.rst

```diff
@@ -189,18 +208,18 @@ that we saw are directly available as model output:
 
    from aospy import Var
 
-    precip_largescale = Var(
-	name='precip_largescale',  # name used by aospy
-	alt_names=('condensation_rain',),  # its possible name(s) in your data
-	def_time=True,  # whether or not it is defined in time
-	description='Precipitation generated via grid-scale condensation',
-    )
-    precip_convective = Var(
-	name='precip_convective',
-	alt_names=('convection_rain', 'prec_conv'),
-	def_time=True,
-	description='Precipitation generated by convective parameterization',
-    )
+   precip_largescale = Var(
+   name='precip_largescale',  # name used by aospy
```

Comment:
I think it would be good to retain the 4 spaces indentation on the continuation lines